data-model/SDP-sample-records/codicological-unit_sample1.json

```diff
@@ -0,0 +1,95 @@
+{
+  "id": "ark:/21198/a310jfBd",
+  "type": "Codicological Unit",
+  "label": "Sinai Syriac 2A, ff. 1-152",
+  "unit_start": {
+    "parent_id": "ark:/21198/z1x07bdf",
+    "locus": "1"
+  },
+  "unit_end": {
+    "parent_id": "ark:/21198/z1x07bdf",
+    "locus": "152"
+  },
+  "origin": {
+    "date_human-readable": "6th c. CE",
+    "date_normalized": "0500/0600",
+    "place": {
+      "authority_file": "https://sinaimanuscripts/library.ucla.edu/place/1",
+      "attested_name_NS": "ܛܘܪ ܣܝܢܝ",
+      "attested_name": "Mount Sinai",
+      "source": "Attested in the colophon on f. 152v"
+      }
+  },
+  "associated_persons":
+    [{
+      "authority_file": "https://sinaimanuscripts/library.ucla.edu/person/1",
+      "attested_name_NS": "ܝܥܩܘܒ ܒܪܐ ܕܒܣܝܠ",
+      "attested_name": "Ya'qub bar d-Basil",
+      "souce": "Attested in the note on f. 114v"
```

Comment:
reading the note on 114v again, this note is not a doxology, certainly, but rather seems to be a (later) readers' note? This does raise the question of how we deal with these kinds of notes, and where information about readers and users should go.

Reply:
Maybe notes like these could go into a 'new-content' (e.g., marginalia, notes, etc.) kind of field. Different levels of paratextual material -- degrees of connection to the text. (level 1 = scribe; level 2 = dedications, etc. that could be later; level 3 = readers comments; level 4 = associated by being in there, e.g. fly-leaf). This is a jumping off point for thinking about these things.

Reply:
Short-term: add a feature, document in contents note with a findable keyword.

Reply:
current Idea is to use 'paratext' at the artifact and part level. The artifact level will contain paratexts that connect thematically with the content of the artifact; the part will contain paratexts that connect or deal in some way with the book (unit) as a produced or migrating object (e.g., colophons, reader's notes, etc.). Definitely should return to this discussion.